tests/test_current_inventory.py

```diff
@@ -14,25 +13,31 @@ def make_beer(quantity):
 
 
 def make_queries(filename):
+    """Tests that a data file can be turned into an InventoryQuery"""
```

Comment:
This one isn't actually testing anything, not sure whether it needs a docstring but if so it should probably change to better fit the functionality